metadata/release/sample.json

```diff
@@ -15,18 +31,84 @@
             }
           },
           "images": {
-            "us-east-1": {
-              "image": "ami-0506465824cb1b578"
+            "ap-northeast-1": {
+              "image": "m-6wedcb2rfmhkcl2bsbz5"
```

Comment:
This is an entry for Aliyun images nested under `aws`; it looks like you have them swapped

I would also update this region to be `us-east-1` for consistency

Reply:
yes, looks like I have swapped them . I will make the change